crates/codegen/schema/src/validation/ast/utils.rs

```diff
@@ -13,7 +13,7 @@ impl cst::Node {
         };
     }
 
-    pub fn get<'a>(&'a self, key: &str) -> &'a cst::NodeRef {
+    pub fn value_of_field<'a>(&'a self, key: &str) -> &'a cst::NodeRef {
```

Comment:
actually I prefer `value_of_field`, because the field name appears right after the word `field`, so it reads like `value of field "name"`. Purely aesthetic.

Reply:
Sounds good.